### PR TITLE
Remove required dependency of GitHelper from Blockchain module

### PR DESCRIPTION
**Pull request by awnion**

### CODE REVIEW

File: git-remote-gosh/src/blockchain/client.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (andrew-kurochkin) [inline git-remote-gosh/src/blockchain/client.rs:1]: seems like mistakely added

File: git-remote-gosh/src/blockchain/contract.rs [hunk 2]
```diff
@@ -37,32 +24,30 @@ pub trait ContractRead: Debug {
...
-        Ok(serde_json::from_value::<T>(result).map_err(|e| anyhow::Error::from(e))?)
-    }
+        for<'de> T: Deserialize<'de>;
 }
 
 #[async_trait]
-pub trait ContractStatic: Debug {
-    async fn static_method<T>(
+pub trait ContractMutate: Debug {
+    async fn mutate_stte<T>(
```
Comment (andrew-kurochkin) [inline git-remote-gosh/src/blockchain/contract.rs:40]: state

File: git-remote-gosh/src/blockchain/client.rs (hunk 1 above)
Comment (andrew-kurochkin) [inline git-remote-gosh/src/blockchain/client.rs:1]: seems like added by mistake

File: git-remote-gosh/src/blockchain/contract.rs (hunk 2 above)
Comment (andrew-kurochkin) [inline git-remote-gosh/src/blockchain/contract.rs:40]: state

File: git-remote-gosh/src/blockchain/commit/save.rs [hunk 3]
```diff
@@ -24,76 +24,80 @@ pub struct DeployCommitParams {
...
-    branch: &str,
-) -> anyhow::Result<()> {
-    let mut buffer: Vec<u8> = Vec::new();
-    let commit = context
-        .local_repository()
-        .objects
-        .try_find(commit_id, &mut buffer)?
-        .expect("Commit should exists");
+#[async_trait]
+pub trait BlockchainCommit: BlockchainService + BlockchainUserWallet {
```
Comment (andrew-kurochkin) [inline git-remote-gosh/src/blockchain/commit/save.rs:28]: why?
- How is that BlockchainCommit if it accepts commits?
- Why it needs Blockchain Service and BlockchainUserWallet?

File: git-remote-gosh/src/lib.rs [hunk 4]
```diff
@@ -12,6 +12,9 @@ extern crate serde_json;
 extern crate base64;
 extern crate base64_serde;
 
+#[macro_use]
+extern crate derive_builder;
```
Comment (andrew-kurochkin) [inline git-remote-gosh/src/lib.rs:16]: How is this used?

**Review by andrew-kurochkin — CHANGES_REQUESTED** [review]

File: git-remote-gosh/src/lib.rs (hunk 4 above)
Comment (awnion) [reply git-remote-gosh/src/lib.rs:16]: TLDR: autogenerate xBuilder for x struct

**Review by andrew-kurochkin — APPROVED** [review]

**Pull request merged**